integrations/gcp/gcp_core/utils.py

```diff
@@ -72,16 +77,48 @@ def get_current_resource_config() -> (
 
 
 def get_credentials_json() -> str:
-    b64_credentials = ocean.integration_config["encoded_adc_configuration"]
-    credentials_json = base64.b64decode(b64_credentials).decode("utf-8")
+    credentials_json = ""
+    if ocean.integration_config["encoded_adc_configuration"]:
+        b64_credentials = ocean.integration_config["encoded_adc_configuration"]
+        credentials_json = base64.b64decode(b64_credentials).decode("utf-8")
+    else:
+        try:
+            file_path: str = (
+                os.environ.get("GOOGLE_APPLICATION_CREDENTIALS")
+                or DEFAULT_CREDENTIALS_FILE_PATH
+            )
+            with open(file_path, "r", encoding="utf-8") as file:
+                credentials_json = file.read()
+        except FileNotFoundError as e:
+            raise FileNotFoundError(
+                f"Couldn't find the google credentials file. Please set the GOOGLE_APPLICATION_CREDENTIALS environment variable or place the credentials file at ~/.config/gcloud/application_default_credentials.json. Error: {str(e)}"
```

Comment:
is `~/.config/gcloud/application_default_credentials.json` something that the user can actually set?